README.md

```diff
@@ -86,8 +95,10 @@ Properties should be provided in camel-casing **without** the data-prefix, e.g.
 | Attribute  | Configuration property | Value type | Description |
 | ---------- | ---------------------- | ---------- | ----------- | 
 | `data-endpoint` | `endpoint` | string | Specifies the starting point of the API that should be rendered. |
+| `data-action-list-position` | `actionListPosition` | string | Specifies the position of the action buttons. Accepted values are: `top` (default), `bottom` and `both`. |
 | | `onValueChange` | function | Callback function executed after a value has been updated. The callback receives 2 parameters: `id` (of the updated element) and `value` (after the change). |
 | | `onUrlChange` | function | Callback function executed after the url has changed. Receives 1 parameter: `url` (after the navigation). |
+| | `onProcessComplete` | function | Callback function executed when the process has ended. Receives 2 parameters: `url` (where the client would normally redirect) and `content` (of that url). If the function exists, the process will NOT automatically redirect. Optional return value: `{ title, content }`. |
```

Comment:
The hypermedia api doesn't really know anything about processes, can we rename this to something more appropriate that is not specific to LM?